recipes/hist/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "hist" %}
+{% set version = "2.0.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/hist-{{ version }}.tar.gz
+  sha256: 48edff8f498ea7c5c6d8057c6f001a2ce394acd0923b921c56b76fb732e11df3
+
+build:
+  number: 0
+  skip: true   # [py2k]
+  entry_points:
+    - hist=hist.classichist:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - boost-histogram ~=0.11.0
```

Comment:
I'm not sure if conda handles the `~=` properly, or if it does it's undocumented and thus not reliable. I think this is the same as `>=0.11.0,<0.12.0`.

Reply:
It is [officially supported](https://github.com/conda/conda/pull/7783) and this receipe was generated by grayskull so if it's a problem grayskull should be modified accordingly.